### DISCUSSION TITLE
Make it possible to import stable maps into Lazer on mobile versions

### DISCUSSION THREAD

**Discussion by WyteHAHA** [ppy/osu#29833] [Ideas] [answered]
I downloaded my entire osu folder and tried to import it but failed

**Comment by Tenexxt** [comment, marked as answer]
I think it's because stable is not Normaly available on Android and iOS (Atleast I couldn't download it for android) so they thought that since no one has it, might as well delete the option so there will be no confusion

**Reply by WyteHAHA** [reply]
> I think it's because stable is not Normaly available on Android and iOS (Atleast I couldn't download it for android) so they thought that since no one has it, might as well delete the option so there will be no confusion

Yes, I get it but there should still be the option for importing it

**Reply by Tenexxt** [reply]
I personally think that on the Setup screen you should have an option to log in/create account after you choose your language (kinda like with PS4) and on your profile you could make packs of maps that you could download if you choose them on the "import/download recommended beatmaps" screen, so you'd have the exact maps you want after the setup.
And before you say you could use favorites for that, this isn't supposed to be used necessarily with favourite beatmaps only, as not every map you like has to be your favorite, or you just didn't add it.

**Reply by WyteHAHA** [reply]
Yeah but that'd probably take a lot of time
